File: scrabble-game/src/main/java/com/aryzko/scrabblegame/application/response/GameStartResponse.java
package com.aryzko.scrabblegame.application.response;

import com.aryzko.scrabblegame.application.model.Player;
import lombok.Builder;
import lombok.Value;

import java.util.List;

@Value
@Builder
public class GameStartResponse {
    private Long id;
    private String boardId;
    private List<Player> players;

    private String actualPlayerId;
    private String winnerId;
}

File: scrabble-ui/src/app/clients/game-manager/model/game-start-response.ts
export interface GameStartResponse {
  id: number;
  boardId: string;
  players: Player[];
}

export interface Player {
  id: string;
  type: Type;
  order: number;
  parameters: Map<string, string>;
}

export enum Type {
  HUMAN,
  BOT,
}
